ipam/http.go

```diff
@@ -199,6 +199,10 @@ func (alloc *Allocator) HandleHTTP(router *mux.Router, defaultSubnet address.CID
 	})
 
 	router.Methods("GET").Path("/ipinfo/tracker").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, alloc.tracker.String())
+		tracker := ""
+		if alloc.tracker != nil {
+			alloc.tracker.String()
```

Comment:
don't you mean `tracker = alloc.tracker.String()`?

Reply:
:man_facepalming: